COPY/usr/bin/manageiq-db-ready

```diff
@@ -26,7 +28,12 @@ def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
 
-  results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
+  # If not present, let psql decide to use localhost, socket, etc.
+  host_arg = host.nil?     ? "" : "--host=#{host}"
+  port_arg = port.nil?     ? "" : "--port=#{port}"
+  pass_arg = password.nil? ? "" : "PGPASSWORD=#{password}"
+
+  results = `#{pass_arg} psql #{host_arg} #{port_arg} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Comment:
Is it imporant that this interpolates the PGPASSWORD into the command line?  When the line didn't have a password then backticks were sufficient, but after adding the password I'm not sure.

I wonder if maybe it's better to use something like Kernel.spawn passing an env or IO or something.

Reply:
I'm not sure.